app/models/menu_items_menu.rb

```diff
@@ -4,7 +4,6 @@ class MenuItemsMenu < ActiveRecord::Base
 
   validates :daily_stock, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :menu_item, presence: true
-  validates :menu, presence: true
```

Comment:
We decided that `menu_items_menus` are always created from the `menu` so we will always have a menu.

A workaround, to keep the validation, would be to add a custom create action on the AA.